common/lib/xmodule/xmodule/util/adaptive_learning.py

```diff
@@ -319,9 +339,16 @@ def get_pending_reviews(self, user_id):
         url = self.pending_reviews_url
         student_uid = self.generate_student_uid(user_id)
         payload = {'student_uid': student_uid, 'nested': True}
-        response = requests.get(url, headers=self.request_headers, data=payload)
-        if response.content == 'No Student Found':
+        content = 'No Student Found'
```

Comment:
@macornwell I see what you're doing here. By defining `content` at this stage, we don't have to access `response.content` twice below.

I'd still prefer to remove this line, as well as L345 (`content = response.content`) below, and reference `response.content` directly in the `else` portion of the try-except-else block:

```python
        url = self.pending_reviews_url
        student_uid = self.generate_student_uid(user_id)
        payload = {'student_uid': student_uid, 'nested': True}
        try:
            response = requests.get(url, headers=self.request_headers, data=payload)
        except requests.ConnectionError:
            pending_reviews_user = []
            log.warning("Could not get pending reviews for user identified by user_id {user_id}.".format(user_id=user_id))
        else:
            if response.content == 'No Student Found':
                pending_reviews_user = []
            else:
                pending_reviews_user = json.loads(response.content)
        return pending_reviews_user
```

**Here's why**: I don't want to mask the fact that there is an inconsistency in the Domoscio API endpoint for obtaining pending reviews. (If the user for which this code tries to obtain pending reviews exists in the Domoscio database, it will return a list of pending reviews. That list might be empty. However, if the user does not exist in the Domoscio database, the API returns a string, "No Student Found", instead of returning an empty list or responding with a 404.) It needs to be clear from reading the code that the "No Student Found" string comes from the Domoscio API. This isn't the case with the current implementation.